job-server-python/src/python/run-tests.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
-PYTHONPATH=.:$SPARK_HOME/python/lib/pyspark.zip:$SPARK_HOME/python/lib/py4j-0.10.7-src.zip:$PYTHONPATH python test/apitests.py
+PYTHONPATH=.:$SPARK_HOME/python/lib/pyspark.zip:$SPARK_HOME/python/lib/py4j-0.10.7-src.zip:$PYTHONPATH \
+PYSPARK_PYTHON=$1 \
+$1 test/apitests.py
```

Comment:
Instead of `$1`, please use the variable.

Reply:
Done